src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -56,6 +56,13 @@ firmware = "@FIRMWAREPATH@"
 # For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
 machine_accelerators="@MACHINEACCELERATORS@"
 
+# Qemu seccomp sandbox feature
+# comma-separated list of seccomp sandbox features to control the syscall access.
+# For example, `seccompsandbox= "on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny"`
+# Please note, enable this feature may lead some performance cost, you may enable
+# /proc/sys/net/core/bpf_jit_enable to reduce the impact.
```

Comment:
Why are we not enabling BPF JIT by default when `-sandbox` is enabled?

Reply:
I suspect this will need to be enabled on the host side, can you confirm @LiangZhou-CTY. 
If so, I dont think we should be enabling this, having this as a recommendation would be the way to go. wdyt @sameo.

Reply:
@amshinde @sameo  yes, BPF JIT configuration is on the host side so I think it's better to leave the decision to the host admin to enable this BPF JIT or not.